dms/models/abstract_dms_mixin.py

```diff
@@ -7,10 +8,13 @@ class AbstractDmsMixin(models.AbstractModel):
     _name = "abstract.dms.mixin"
     _description = "Abstract Dms Mixin"
 
+    # ----------------------------------------------------------
```

Comment:
Although I know you are mimicking it, this comment format is taking 3 lines for only expressing one thing, so it's better to avoid it. The current way of distributing the code doesn't need extra comments IMO, but if you want to put them, just put one line like:
```suggestion
    # FIELDS
```